packages/calcite-components/src/components/color-picker/color-picker.tsx

```diff
@@ -491,9 +498,27 @@ export class ColorPicker
     if (inputValue !== "" && this.shiftKeyChannelAdjustment !== 0) {
       // we treat nudging as a change event since the input won't emit when modifying the value directly
       this.handleChannelChange(event);
+    } else if (inputValue !== "") {
+      this.handleChannelChange(event);
     }
   };
 
+  private handleChannelBlur = (event: CustomEvent): void => {
+    const input = event.currentTarget as HTMLCalciteInputNumberElement;
+    const channelIndex = Number(input.getAttribute("data-channel-index"));
```

Comment:
nitpick: would be nice in the future if we didn't have to convert this to a string in the DOM and then have to parse it back as a Number. Maybe `channelsToRender` could be stored and used to lookup the index in the future.